### PR TITLE
Bases for further programming and fixing NotImplemented Exceptions

### PR DESCRIPTION
**Pull request by greym0uth**
Made it possible for a single emote to have multiple expressions. Job definitions now contain an array for starting items. Fixed NotImplementedException freezing the server.

### CODE REVIEW

**Comment by PJB3005** [comment]
Uh, good track record for atomic PRs so far.

Really though I would recommend to make PRs to do a SINGLE thing and to not bundle it up like this.

**Comment by PJB3005** [comment]
Uh dude I'd recommend if you separated your PRs a bit instead of making a mess like this.

**Comment by Jaden-Giordano** [comment]
Mess? Perhaps we could talk this over in the IRC?

File: SS14.Server.GameObjects/Component/Inventory/InventoryComponent.cs [hunk 1]
```diff
@@ -92,7 +92,7 @@ public override void HandleExtendedParameters(XElement extendedParameters)
         }
 
         //Adds item to inventory and dispatches hide message to sprite compo.
-        // TODO this method should be renamed to reflect what it really does
+        [System.Obsolete("Use RaiseInventoryAdd instead.")]
```
Comment (ComicIronic) [inline SS14.Server.GameObjects/Component/Inventory/InventoryComponent.cs:95]: Did you mark a private non-API obsolete?

**Comment by greym0uth** [comment]
Yeah, there was supposed to be a public method for adding inventory items and it replaced that function.

File: SS14.Client.GameObjects/Component/Inventory/InventoryComponent.cs [hunk 2]
```diff
@@ -72,6 +72,7 @@ public override void HandleComponentState(dynamic state)
             var toRemove = new List<Entity>();
             foreach (var e in ContainedEntities)
             {
+                Console.WriteLine(e.Uid);
```
Comment (psykzz) [inline SS14.Client.GameObjects/Component/Inventory/InventoryComponent.cs:75]: This isn't needed, it should be removed before merged.

**Comment by greym0uth** [comment]
Should close this pull request, there still errors, and I was really inexperienced with the open sourced workflow, will fix and create a new pull request.

**Pull request closed without merging**